src/App.vue

```diff
@@ -1,15 +1,24 @@
 <template>
   <div id="app">
+    <HeaderLayout />
     <router-view></router-view>
   </div>
 </template>
 
 <script lang="ts">
+import HeaderLayout from "./presentation/layout/HeaderLayout.vue";
 export default {
   name: "App",
+  components: {HeaderLayout},
```

Comment:
It's question for @neSpecc, it is layout? So we have several layouts?

Reply:
No, it's just a component

Reply:
@slaveeks rename pls 🙏

Reply:
done